### ISSUE TITLE
Bug:  warning  Property 'outline' is not a widely available baseline feature  css/use-baseline

### ISSUE THREAD

**Issue by budarin** [eslint/css#288]
### Environment

ESLint version: 9.37.0
@eslint/css version: 0.11.1
Node version: 24.8.0
pnpm version: 10.15.1
Operating System: Windows 10 

### Which language are you using?

rule

### What did you do?

eslint config
```js
import css from '@eslint/css';

export default defineConfig([
    globalIgnores(['dist']),
    {
        files: ['src/**/*.css'],
        plugins: {
            css,
        },
        language: 'css/css',
        rules: {
            'css/no-duplicate-imports': 'error',
            'css/use-baseline': [
                'warn',
                {
                    available: 'widely',
                },
            ],
        },
    },
```

styles
```css
button:focus,
button:focus-visible {
    outline: 4px auto -webkit-focus-ring-color;
}
```

hovering text:
```
Shorthand property for 'outline-style', 'outline-width', and 'outline-color'.

Baseline icon Newly available across major browsers (Baseline since 2023)

Syntax: <'outline-width'> || <'outline-style'> || <'outline-color'>
```

### What did you expect to happen?

pass successfully css linting

### What actually happened?

have a wrong warning:
```
warning  Property 'outline' is not a widely available baseline feature  css/use-baseline
```

### Link to Minimal Reproducible Example

no

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

in MDN:
outline is widely available in 2025
